src/index.js

```diff
@@ -22,8 +24,35 @@ var _ = require('lodash'),
 
 var BROWSER_SCIPTS_DIR = path.join(__dirname, 'browser-scripts');
 
+var levels = [
+  'error',
+  'warn',
+  'info',
+  'debug',
+];
+var instanceNumber = 0;
 
+function bindLogger(logLevel, kind) {
+  var logger = glogg('gulp-server-livereload-' + kind + '-' + instanceNumber);
 
+  logLevel = levels.indexOf(logLevel) + 1;
+
+  if (logLevel == 0) {
+    throw 'Logging level "' + logLevel + '" does not exist!';
+  }
+
+  levels
+    .filter(function (item, i) {
+      return i < logLevel;
+    })
+    .forEach(function (level) {
+      logger.on(level, function () {
+        gulpLogger.info.apply(gulpLogger, arguments);
```

Comment:
Surely we want to log to console using the same logging level instead of always using `info`?